calicoctl/commands/docoptutils/validate.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docoptutils
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	cnet "github.com/projectcalico/libcalico-go/lib/net"
+	"github.com/projectcalico/libcalico-go/lib/numorstring"
+)
+
+// validateIP takes a string as an input and makes sure it's a valid IPv4 or IPv6 address.
+func ValidateIP(str string) cnet.IP {
```

Comment:
I was thinking last night that calling this docoptutils was misleading.... I'll rename the package to argutils